src/main/java/com/commercetools/sync/types/TypeSync.java

```diff
@@ -48,6 +48,18 @@ public TypeSync(@Nonnull final TypeSyncOptions typeSyncOptions) {
         this.typeService = new TypeServiceImpl(typeSyncOptions);
     }
 
+    /**
+     * Takes a {@link TypeSyncOptions} and a {@link TypeService} instances to instantiate
+     * a new {@link TypeSync} instance that could be used to sync type drafts in the CTP project specified
+     * in the injected {@link TypeSyncOptions} instance.
+     *
+     * <p>NOTE: This constructor is mainly to be used for tests where the services can be mocked and passed to.
```

Comment:
why is it still public then?

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/347/commits/8a5dd4d91f29b3d01b62e84db6ca2b109c408029